recipe/meta.yaml

```diff
@@ -57,7 +57,9 @@ outputs:
       build:
         - libprotobuf {{ protobuf }}
         - grpc-cpp {{ grpc_cpp }}
-        - cmake 3.19   # to avoid https://gitlab.kitware.com/cmake/cmake/-/issues/18810
+        # to avoid https://gitlab.kitware.com/cmake/cmake/-/issues/18810
+        - cmake 3.19        # [not s390x]
+        - cmake {{ cmake }} # [s390x]
```

Comment:
Only cmake 3.18.2 is available on s390x. So using that version.

Reply:
This is fine - we should be able to clean this up eventually.